src/components/dashboard/dashboard.component.tsx

```diff
@@ -105,6 +111,9 @@ function FormsList({ forms, isValidating, t }) {
   const isTablet = useLayoutType() === "tablet";
   const [filter, setFilter] = useState("");
   const config = useConfig();
+  const [pageSize, setPageSize] = useState(10);
+  const pageSizes = config?.activeVisits?.pageSizes ?? [10, 20, 30, 40, 50];
```

Comment:
Am not so inclined to using the context of active visits in the form builder. Maybe we could just use the array and eliminate the `config?.activeVisits?.pageSizes`